### PR TITLE
Fixing empty bug

### PR DESCRIPTION
**Pull request by John42506176Linux**
DONT LOOK AT THIS PR Until CLoud Functions Update gets approved and merged.
This is to mainly to fix a problem with tickers returning null occasionally. The database was shortened because of it. No screenshot because it's not a visual change.

### CODE REVIEW

File: src/components/firebase-access.jsx [hunk 1]
```diff
@@ -130,31 +130,39 @@ export const initializeQuiz = async (symbols, roomId, periodLen, endDates) => {
...
-        //Comment this line back out when you want to run on localhost and add localhost, and comment out headers
-        headers: {
-          Authorization: ("Bearer " + token)
-        },
+        //Comment this line back in when you want to deploy , and get rid of localhost
+        // headers: {
+        //   Authorization: ("Bearer " + token)
+        // },
       },100000)
-      console.log(response); // This is here until the next pr gets approved.
```
Comment (asam01) [inline src/components/firebase-access.jsx]: can you set all this stuff back to what it was on master?

File: src/components/firebase-access.jsx [hunk 2]
```diff
@@ -130,31 +130,39 @@ export const initializeQuiz = async (symbols, roomId, periodLen, endDates) => {
...
-          //Comment this line back out when you want to run on localhost and add localhost, and comment out headers
-          headers: {
-            Authorization:("Bearer " + token)
-          }
+          //Comment this line back in when you want to deploy , and get rid of localhost
+          // headers: {
+          //   Authorization:("Bearer " + token)
+          // }
       },100000)
-      console.log(response); // Same as above
```
Comment (asam01) [inline src/components/firebase-access.jsx]: also set this back to align with master

File: src/components/firebase-access.jsx [hunk 3]
```diff
@@ -177,14 +185,14 @@ export const initSymbols = async(industry,sector,marketCap,numOfSymbols) =>{
...
               body: formData,
-              //Comment this line back out when you want to run on localhost and add localhost, and comment out headers
-              headers: {
-                Authorization: (' Bearer ' + token)
-              }
+              //Comment this line back in when you want to deploy , and get rid of localhost
+              // headers: {
+              //   Authorization: (' Bearer ' + token)
+              // }
           },100000)
```
Comment (asam01) [inline src/components/firebase-access.jsx]: also restore to master version

File: src/components/pages/Host.jsx [hunk 4]
```diff
@@ -40,6 +40,8 @@ class Host extends Component {
             authenticated: 'no',
             listening: 'no',
             users: [],
+            images: null,
+            prices: null,
```
Comment (asam01) [inline src/components/pages/Host.jsx]: why does the host need to store images and prices?

File: stock_backend/StoreTickerData.py [hunk 5]
```diff
@@ -21,43 +22,78 @@
...
+# if path[-1]  is '/':
+#     path[:-1]
+
+# webbrowser.open("https://old.nasdaq.com/screening/companies-by-name.aspx?letter=0&exchange=nasdaq&render=download")
+# time.sleep(3)
+NASDAQ =  pd.read_csv("companylistNASDAQ.csv")
+# os.remove(path +"/companylist.csv")
+# webbrowser.open("https://old.nasdaq.com/screening/companies-by-name.aspx?letter=0&exchange=nyse&render=download")
+# time.sleep(3)
+NYSE =  pd.read_csv("companylistNYSE.csv")
```
Comment (asam01) [inline stock_backend/StoreTickerData.py]: what is all of this stuff that's commented out?

File: stock_backend/StoreTickerData.py [hunk 6]
```diff
@@ -21,43 +22,78 @@
...
+StockEx = StockEx.dropna(subset=['IPOyear'])
+StockEx = StockEx.drop_duplicates()
+# We want at least 2 years worth of data to work with not including the present year
+# So we always delete data before that.
+IPOCutoff = datetime.date.today() - relativedelta(years=3)
+StockEx =  StockEx[StockEx.IPOyear <= IPOCutoff.year]
+num = 0
+size = len(StockEx.index)
+for (index,stock) in StockEx.iterrows(): 
+        num+=1
```
Comment (asam01) [inline stock_backend/StoreTickerData.py]: keep indentation consistent. this is 8 spaces - before, you used 4

**Review by asam01 — CHANGES_REQUESTED** [review]
left some comments

**Comment by John42506176Linux** [comment]
The 6000 lines is because I added the csv files to the commit, its around 200-300 lines of code

**Comment by John42506176Linux** [comment]
I also added the gray out button to the commit just to make i simple.

**Review by asam01 — COMMENTED** [review]
lgtm, can you just fix the try/catch style to include a space after try?

`try {`

**Review by asam01 — APPROVED** [review]

File: src/components/firebase-access.jsx [hunk 7]
```diff
@@ -154,7 +166,18 @@ export const initializeQuiz = async (symbols, roomId, periodLen, endDates) => {
...
+      event.initEvent("storage", true, true);
+      if(response.status == 200) {
+        localStorage.setItem('Images', 'Received');
+      }
+      else if(response.status == 401) {
+        localStorage.setItem('Images', 'Unauthenticated');
+      }
+      else if(response.status == 500) {
+        localStorage.setItem('Images', 'Error');
+      }
```
Comment (yukyhaha) [inline src/components/firebase-access.jsx:179]: what if some other error code returned?

**Review by yukyhaha — APPROVED** [review]

**Pull request merged**